__tests__/app.test.js

```diff
@@ -36,6 +36,32 @@ describe("GET /api", () => {
     })
 })
 
+describe("GET /api/articles", () => {
+    test("status: 200 responds with array of articles, with comment_count column added and populated correctly, sorted by date, descending order", async () => {
+        const response = await request(app).get("/api/articles")
+        expect(response.status).toBe(200)
+        response.body.articles.forEach((article) => {
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String),
+            })
+        })
+        expect(response.body.articles).toBeSortedBy('created_at', { descending: true })
+    })
+    test("status: 404 for bad routes (get request for non existent data)", async () => {
+        const response = await request(app).get("/api/gibberish")
+        expect(response.status).toBe(404)
```

Comment:
This wouldn't be a 404, it would be a 400 as it's an invalid data type. There should also be a 404 test as well though for something like `/api/9999 `